src/main/java/core/basesyntax/Models.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Models {
```

Comment:
It's recommended to use singular names for enum types. Rename `Models` to `Model` 🚀